specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.ApiManagement/ApiManagement.tsp

```diff
@@ -130,6 +130,70 @@ model ApiManagementGatewayApiRemovedEventData {
   ...ApiGatewayApiBase;
 }
 
+/** Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.CircuitBreaker.Opened event. */
+model ApiManagementCircuitBreakerOpenedEventData {
+  /** Name of the backend for which the circuit has opened. */
+  backendName: string;
+
+  /** Information related to the circuit breaker configured on the backend. */
+  circuitBreaker: ApiManagementCircuitBreakerProperties;
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.CircuitBreaker.Closed event. */
+model ApiManagementCircuitBreakerClosedEventData {
+  /** Name of the backend for which the circuit has closed. */
+  backendName: string;
+
+  /** Information related to the circuit breaker configured on the backend. */
+  circuitBreaker: ApiManagementCircuitBreakerProperties;
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.GatewayTokenNearExpiry event. */
+model ApiManagementGatewayTokenNearExpiryEventData {
+  /** Information related to a given self-hosted gateway deployment. */
+  gatewayInfo: ApiManagementGatewayProperties;
+
+  /** Information related to a an expired gateway token for a self-hosted gateway deployment. */
+  tokenInfo: ApiManagementNearExpiryGatewayTokenProperties;
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.GatewayTokenExpired event. */
+model ApiManagementGatewayTokenExpiredEventData {
+  /** Information related to a given self-hosted gateway deployment. */
+  gatewayInfo: ApiManagementGatewayProperties;
```

Comment:
Can we also rename the properties? This can probably just be `gateway`.

Reply:
Done for expired & near expiry